Package.swift

```diff
@@ -1,7 +1,15 @@
-// swift-tools-version:3.1
+// swift-tools-version:5.1
 
 import PackageDescription
 
 let package = Package(
-    name: "UInt128"
+    name: "UInt128",
+    products: [
+        .library(name: "UInt128", targets: ["UInt128"]),
+    ],
```

Comment:
Is the definition of the products parameter to Package with a library name of "UInt128" the reason why you moved the source code file to ./Sources/UInt128/ instead of keeping it directly beneath ./Sources/?

Reply:
Yes, the default path is `[PackageRoot]/Sources/[TargetName]`.

There is a `path` parameter available: [docs.swift.org/package-manager/PackageDescription/PackageDescription.html](https://docs.swift.org/package-manager/PackageDescription/PackageDescription.html#target). It may be useful if you want to keep the original simple path `[PackageRoot]/Sources`.

Reply:
Thanks for that description! I'm good with this change if it keeps us in sync with current best practices.